esti/ops/docker-compose-dynamodb.yaml

```diff
@@ -20,18 +31,51 @@ services:
       - LAKEFS_LOGGING_LEVEL=DEBUG
       - LAKEFS_BLOCKSTORE_GS_CREDENTIALS_JSON
       - LAKEFS_STATS_ENABLED
-      - LAKEFS_DATABASE_DYNAMODB_ENDPOINT=http://dynamodb:8000
       - LAKEFSACTION_VAR=this_is_actions_var
       - LAKEFS_DATABASE_TYPE=dynamodb
-      - LAKEFS_DATABASE_KV_ENABLED=true
+      - LAKEFS_DATABASE_DYNAMODB_ENDPOINT=http://dynamodb:8000
       - LAKEFS_DATABASE_DYNAMODB_AWS_REGION=us-east-1
       - LAKEFS_DATABASE_DYNAMODB_AWS_ACCESS_KEY_ID=AKIAIO5FODNN7EXAMPLE
       - LAKEFS_DATABASE_DYNAMODB_AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K3MDENG/bPxRfiCYEXAMPLEKEY
-    entrypoint: ["/app/lakefs", "run"]
+    entrypoint: ["/app/wait-for", "fluffy:8000", "--", "/app/lakefs", "run"]
+
+  fluffy:
+    # TODO (niro): replace with 'latest'
```

Comment:
Please don't: that would make tests non-repeatable.  And that _increases_ the likelihood of breaking an enterprise customer.

If we must, we can add a matrix of versions to test.

Reply:
What do you mean - that makes tests not repeatable?
We are not testing the fluffy server. We are testing lakeFS.
I'm not sure that this is the place to perform the this discussion in

Reply:
Replacing with `latest` means that tests may suddenly fail/pass due to Fluffy version bump. I agree that we should pin the version and that the `TODO` is redundant.

Reply:
Done